app/build.gradle

```diff
@@ -60,6 +62,7 @@ android {
             resValue("string", "firebase_app_id", FIREBASE_APP_ID_DEBUG ?: "")
         }
         master {
+            // TODO: look into this build variant, is it really needed?
```

Comment:
Forgot about it.
I have just checked it and the only place it is being used is by Bitrise CI. In most workflows, it is being built on the `debug` variant. But there are 2 that use `master` for some unexplained reason:

- [develop workflow](https://github.com/salemove/android-sdk-widgets/blob/a7be117886c6d9ad86e078105c490f051e35b0d0/bitrise.yml#L184) -> not used anymore (before was triggered by changes in 'develop' branch)
- [master workflow](https://github.com/salemove/android-sdk-widgets/blob/a7be117886c6d9ad86e078105c490f051e35b0d0/bitrise.yml#L240) -> triggered by changes in 'master' branch

But I guess that it is safe to just use the 'debug' build type instead for all workflows especially since it basically just changed the package name to have suffix ".master"

Reply:
`master` variant also has a different `firebase_app_id` - `FIREBASE_APP_ID_MASTER`. But I don't think that it's needed.